### ISSUE TITLE
[sim-services] Documentation is horrible (Trac #998)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#992]
Most modules are not documented at all.  This needs to be fixed before the next release.

Migrated from https://code.icecube.wisc.edu/ticket/998
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "Most modules are not documented at all.  This needs to be fixed before the next release.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo simulation", 
    "summary": "[sim-services] Documentation is horrible", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-05-26T21:32:53", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150811T00:25:27: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150909T20:15:46: 
* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 137416:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137416"
> i'd call this a significant improvement in the docs.  at least they're not terrible anymore. closes #992.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas changed milestone from "IceSim 4.2" to ""
* olivas commented:

> Milestone IceSim 4.2 deleted
